src/v/cluster/metrics_reporter.cc

```diff
@@ -262,6 +270,10 @@ metrics_reporter::build_metrics_snapshot() {
         snapshot.id_hash = license->checksum;
     }
 
+    snapshot.has_valid_license = license.has_value()
+                                 && license.value().is_expired();
+    snapshot.has_enterprise_features = feature_report.any();
+
```

Comment:
I think the commit from the fix PR is missing here: https://github.com/redpanda-data/redpanda/pull/23622/commits/998258899788757e19452b14d42f2003dc85f4f4

Reply:
ah right, totally forgot about that. great catch, thanks!

Reply:
done